README.md

```diff
@@ -1,3 +1,25 @@
 ![](assets/party.gif)
 # Eggy-Bowl
 This is a Egg catcher Game. I designed and developed it. I do represent it with name "Eggy-Bowl". It is a 3-D graphical game using Multiple python libraries.
+
+This Game can be played using arrow keys.
+
+Left arrow key : to move catcher left
+
+Right arrow key : to move catcher right
+
+Built With
+### Python
+### tkinter
+### itertools
+### random
+
+Getting Started
+### To play download .exe and play.
+
+### To create your own download python and getstarted.
+
+Installation
+### Clone the repo
+
+git clone https://github.com/rituvermaCS/Eggy-Bowl
```

Comment:
I believe that this would fall under the previous subdivision, and is thus unnecessary

Reply:
Yeah !! that's right, I guess I should have removed it. But no worries I'll keep that in mind, next time.